lib/PageAgent.js

```diff
@@ -46,6 +46,14 @@ extend(PageAgent.prototype, {
       this.emit('resource-tree');
     }.bind(this);
 
+    //skip building a list of source files from the filesystem
```

Comment:
`// skip...` (notice the missing space after double-slash, which is inconsistent with the next line)